lib/chirp_cockroach_web/user_auth.ex

```diff
@@ -94,6 +94,20 @@ defmodule ChirpCockroachWeb.UserAuth do
     assign(conn, :current_user, user)
   end
 
+  def fetch_api_current_user(conn, _opts) do
+    with ["Bearer " <> token] <- get_req_header(conn, "authorization"),
+         {:ok, user} <- Accounts.fetch_user_by_api_token(token) do
+      conn
+      |> assign(:current_user, user)
+      |> Absinthe.Plug.put_options(context: %{current_user: user})
+    else
+      # :error ->
```

Comment:
We could log a warning here, that the app is allowing access to an anonymous user.